bin/pgenv

```diff
@@ -177,44 +269,57 @@ case $1 in
         fi
 
         # Start er up!
-        $PG_CTL start -l $PG_DATA/server.log
-        echo "PostgreSQL $v started"
+        trap "" ERR
+        $PG_CTL start -w -l $PG_LOG
+        if [ $? -eq 0 ]; then
+            echo "PostgreSQL $v started"
+            echo "Logs are in [$PG_LOG]"
```

Comment:
Don't think the brackets are necessary here.

Reply:
In fact, that's always the location fo the logs, so seems overly verbose to me.

Reply:
Yes, but the logs are not in a standard place, so I guess there is no problem repeating where the logs are (and besides, this would allow us to move logs without having to inform the user thru the documentation).